File: 771-jewels-and-stones/771-jewels-and-stones.py
class Solution:
    def numJewelsInStones(self, jewels: str, stones: str) -> int:
        count = 0
        freq = [False]*58
        for i in jewels:
            freq[ord(i) - 65] = True
        for i in stones:
            if(freq[ord(i)-65]): count+=1
        return count

File: 771-jewels-and-stones/771-jewels-and-stones.cpp
class Solution {
public:
    int numJewelsInStones(string jewels, string stones) {
        if (jewels.length() == 0){ return 0; } 
        if (stones.length() == 0){ return 0;}
        int count=0;
        for (int i =0 ;i<jewels.length();i++){
            for (int j =0 ; j<stones.length();j++){
                if (jewels[i]==stones[j]){
                    count++;
                }
            }
            
        }
        return count;
        
    }
};